services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -226,6 +231,19 @@ def log_level(self) -> int:
     def valid_log_level(cls, value) -> str:
         return cls.validate_log_level(value)
 
+    @validator("SC_HEALTHCHECK_TIMEOUT", pre=True)
+    @classmethod
+    def get_healthcheck_timeout_in_seconds(cls, v):
+        # Ex. HEALTHCHECK --interval=5m --timeout=3s
+        if isinstance(v, str):
+            if v.endswith("s"):
+                factor = 1
+            elif v.endswith("m"):
+                factor = 60
+            v = v.rstrip("ms")
```

Comment:
The default config gets confused with "m" but understands "ms" 
```cmd
>>> from pint import UnitRegistry
>>> ureg = UnitRegistry()
>>> ureg.Quantity("3m")
<Quantity(3, 'meter')>
>>> ureg.Quantity("3ms")
<Quantity(3, 'millisecond')>
>>> 
```

Reply:
Ok, instead of adding custom code to handle timekeeping units I just decided I would prefer to have everything throughtout simcore in milliseconds, even if it implies large numbers. I appreciate the effort but manual time-unit conversion is an avoidable potential error-source

Reply:
here there is no risk. It is only one variable and we set it.